src/ConfigurationRemotingServer/Program.cs

```diff
@@ -21,6 +31,73 @@ static int Main(string[] args)
 
                 PowerShellConfigurationSetProcessorFactory factory = new PowerShellConfigurationSetProcessorFactory();
 
+                // Set default properties.
+                var externalModulesPath = GetExternalModulesPath();
+                if (!string.IsNullOrWhiteSpace(externalModulesPath))

```

Comment:
This case should probably result in an error rather than silently skipping the external modules path.